bedrock/externalpages/templates/externalpages/pocket/pocket-and-firefox.html

```diff
@@ -0,0 +1,102 @@
+<section class="explore-stories">
+  <div class="explore-stories-wrapper mzp-l-content mzp-t-content-md">
+    <h2 class="sub-section-heading">Get thought-provoking article recommendations</h2>
+    <p class="section-lede">In addition to saving stories from all over, you can also dig into the latest articles we're recommending in Pocket. Our editors find seriously interesting stuff — stories that dig deep into a subject, offer a new perspective, or make you think.</p>
+    <a href="https://getpocket.com/explore" class="discover-cta" role="button">Discover stories in Pocket</a>
+  </div>
+</section>
+
+<section>
+  {% call split(
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/reading-nook.png',
+  )%}
+  <h2 class="sub-section-heading">Your private reading nook</h2>
+  <p class="section-lede">When you’re ready to dig into what you’ve saved, Pocket provides a quiet, calm space that’s perfect for reading. It strips away all the distractions of the internet — like flashing banners and pop-ups — so you can really focus.</p>
+  {% endcall %}
+</section>
+
+<section>
+  {% call split(
+  block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/focused-reading.png',
+  )%}
+  <h2 class="sub-section-heading">Special features for focused reading</h2>
+  <p class="section-lede">Use the Pocket app to listen to stories with our audio playback option. Customize how you see stories in Pocket with dark mode and font options. Keep track of your saves with tags, and preserve important passages with highlights. You can even read offline.</p>
+  {% endcall %}
+</section>
+
+
+  <section>
+  {% call split(
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/save-button-doodle.svg',
+  )%}
+  <h2 class="sub-section-heading">Your save button for the internet</h2>
+  <p class="section-lede">Click the <span class="pocket-logo-inline">Pocket Logo</span> button in Firefox anytime you see something you’d like to save, wherever you are online. It’ll go straight to your list in Pocket, ready for you to dig into when you’re free.</p>
+  {% endcall %}
+</section>
+
+<section class="personal-library">
+  <h2 class="section-heading">
+    Start building your personal library now
+  </h2>
+  <div class="library-ctas">
+    {{ pocket_fxa_button(entrypoint='pocket', class_name='new-tab-pocket-cta', button_text='<span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family"></span>Activate Pocket in Firefox', optional_parameters={'s': 'fflearnmore', 'utm_campaign': 'landing-page', 'utm_content': 'page-button'}, optional_attributes={'data-cta-text': 'Activate Pocket', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}
+    <a href="https://getpocket.com/explore" class="discover-cta" role="button">Discover stories in Pocket</a>
+  </div>
```

Comment:
**NIt**: There should be another link below the pocket cta - `More ways to Save`


<img width="875" alt="Screen Shot 2021-12-23 at 9 50 26 AM" src="https://user-images.githubusercontent.com/30009669/147275954-3b92806e-6b26-4a56-8616-5e572a028f80.png">